package.json

```diff
@@ -40,6 +41,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "eslint --ignore-path .gitignore --ext .js,.jsx ."
+    "lint": "eslint --ignore-path .gitignore --ext .js,.jsx .",
+    "format": "eslint --ignore-path .gitignore --fix --ext .js,.jsx ."
```

Comment:
You put the `format` back. I tried to avoid it because with yarn it's possible to use it as simple as `yarn lint --fix` but `npm` does things completely weird. Although, `npm run link -- --fix` could work (I did not test it enough times).

Reply:
I wondered where did it go. Sorry I get it now.